run_daisy_chain/daisy_chain_env.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Arg parse library, or others add a really nice user interface from the `-h` flag, so we should definitely use that and remove this.